docs/source/using-diffusers/loading.mdx

```diff
@@ -12,7 +12,347 @@ specific language governing permissions and limitations under the License.
+```
+
+Therefore, we need to make sure to *click-accept* the license as well as run:
+
+```
+huggingface-cli login
+```
+
+before trying to load the model. Or alternatively, you can pass [your access token](https://huggingface.co/docs/hub/security-tokens#user-access-tokens) directly via the flag `use_auth_token`. In this case you do **not** need 
+to run `huggingface-cli login` before:
+
+```python
+from diffusers import DiffusionPipeline
+
+repo_id = "runwayml/stable-diffusion-v1-5"
+stable_diffusion = DiffusionPipeline.from_pretrained(repo_id, use_auth_token="<your-access-token>")
+```
+
+The final option to circumvent having to be logged in to the Hugging Face Hub to use pipelines that require access is to load the locally as explained in the next section.
+
+### Loading pipelines locally
+
+If you prefer to have complete control over the pipeline and its corresponding files or, as said before, if want to use pipelines that require access requests without having to be connected to the Hugging Face Hub in one way or 
+another, we recommend loading pipelines locally.
+
+To load pipeline locally, you first need to manually download the whole folder structure on your local disk and then pass a local path to the [`DiffusionPipeline.from_pretrained`]. Let's again look at an example here for 
+[CompVis' Latent Diffusion model](https://huggingface.co/CompVis/ldm-text2im-large-256).
+
+First, you should make use of [`git-lfs`](https://git-lfs.github.com/) to download the whole folder structure that has been uploaded to the model repository: https://huggingface.co/CompVis/ldm-text2im-large-256/tree/main via:
+
+```
+git lfs install
+git clone https://huggingface.co/CompVis/ldm-text2im-large-256
+```
+
+The command above will create a local folder called `./ldm-text2im-large-256` on your disk.
+Now, all you have to do is to simply pass the local folder path to `from_pretrained`:
+
+```python
+from diffusers import DiffusionPipeline
+
+repo_id = "./ldm-text2im-large-256"
+stable_diffusion = DiffusionPipeline.from_pretrained(repo_id)
+```
+
+In case `repo_id` is a local path as it is the case here [`DiffusionPipeline.from_pretrained`] will automatically detect it and therefore not try to download any files from the Hub.
+While we usually recommend to load weights directly from the Hub to be certain to stay up to date with the newest changes, loading pipelines locally should be preferred if one 
+**wants to stay anonymous** or disconnected from the internet.
+
+### Loading customized pipelines
+
+Advanced users that want to load customized versions of diffusion pipelines can do so easily by swapping any of the default components, *e.g.* the scheduler, with other scheduler classes.
+For example, [Stable Diffusion v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5) uses the [`PNDMScheduler`] by default which is generally not the most performant scheduler. Instead since the release
```

Comment:
```suggestion
A classical use case of this functionality is to swap the scheduler. [Stable Diffusion v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5) uses the [`PNDMScheduler`] by default which is generally not the most performant scheduler. Since the release
```